### ISSUE TITLE
Dynamic Column Family Creation for Commit and Error logs

### ISSUE THREAD

**Issue by boneill42** [hmsonline/cassandra-triggers#45]
I have a question about the schema, and rather than raising a ticket for what might be a silly error on my part, I thought I'd just email you instead.

As I noted in issue 42 (https://github.com/hmsonline/cassandra-triggers/issues/42), the Wiki page about configuration seems to be out of date with regard to the CF names. I'm therefore unsure what CFs I have to add to Cassandra's "triggers" keyspace.  I added a CF called "CommitLog" as per the Wiki, but it's empty even after I do some inserts to Cassandra. From looking at your code, specifically the LogEntryStore constructor, it seems that the commit and error log CFs should have the host name appended to them. But I'm still unclear whether your code will create these CFs for me or whether I need to create them myself. At the moment I'm getting this exception in my Cassandra console (where "my_host_name" is my workstation's host name with any illegal characters replaced):

InvalidRequestException(why:unconfigured columnfamily CommitLog_my_host_name)
    at org.apache.cassandra.thrift.ThriftValidation.validateColumnFamily(ThriftValidation.java:129)
    at org.apache.cassandra.thrift.CassandraServer.multigetSliceInternal(CassandraServer.java:341)
    at org.apache.cassandra.thrift.CassandraServer.get_slice(CassandraServer.java:326)
    at com.hmsonline.cassandra.triggers.dao.CommitLog.getPending(CommitLog.java:92)
    at com.hmsonline.cassandra.triggers.TriggerTask.run(TriggerTask.java:47)
    at java.lang.Thread.run(Thread.java:722)

Ideally your code would create this CF on the fly, otherwise the administrator has to (a) determine the sanitised host name for the Cassandra server and then (b) create that CF manually.

**Comment by boneill42** [comment]
We went back and forth on this, which is probably why it isn't clear
in the code.  In a cluster, when we created the column families
dynamically, we would occasionally see schema disagreement issues.  We
decided to pull that code out until Cassandra supported run-time
changes to the schema.

Now that Cassandra has better support for dynamic schema changes, we
can probably re-introduce the column family creation.
You'll see CassandraStore.create() will go ahead and create the column
family.  We just need to catch the exception and call that method.

Per the previous comment RE: dynamic schemas, it is probably worth
upgrading to 1.1 at the same time.  I'll create an issue for this.  If
you have time, I'd gladly take a pull request. =)  If not, I can
probably get to it next week.
